packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -264,72 +255,7 @@ const Injector = ({ children }: { children: React.ReactNode }): JSX.Element => {
       async function logGasPrice() {
```

Comment:
I think we can get rid of this entire effect now, as the only thing it does is log gas price to the console.